docs/src/main/sphinx/security/jwt-auth.rst

```diff
@@ -0,0 +1,145 @@
+       PEM or HMAC file, as described below this table.
+   * - ``http-server.authentication.jwt.required-issuer``
+     - Optional. Specifies a string that must match the value of the JWT's
+       ``iss`` field to consider this JWT valid for this Trino server.
+       The ``iss`` field in the JWT identifies the principal that issued the
+       JWT.
+   * - ``http-server.authentication.jwt.required-audience``
+     - Optional. Specifies a string that must match the value of the JWT's
+       ``aud`` field to consider this JWT valid for this Trino server.
+       The ``aud`` field in the JWT identifies the recipients that the JWT is
+       intended for.
+   * - ``http-server.authentication.jwt.principal-field``
+     - Optional. String to identify the field in the JWT that identifies the
+       subject of the JWT. The default value is ``sub``.
+   * - ``http-server.authentication.jwt.user-mapping.pattern``
+     - Optional. A regular expression pattern to map all user names for this
+       authentication system to the format expected by the Trino server. See
+       :doc:`User mapping </security/user-mapping>`.
+   * - ``http-server.authentication.jwt.user-mapping.file``
+     - Optional. The path to a JSON file that contains a set of user mapping
+       rules for this authentication system. See :doc:`User
+       mapping </security/user-mapping>`.
+
+Use the ``http-server.authentication.jwt.key-file`` property to specify
+either:
+
+-  The URL to a JWKS endpoint service, where the URL begins with ``https://``
+   and ends with the port number. The JWKS service must be reachable from the
+   coordinator. If the coordinator's local network locks down port access for
+   security, then administrators must either open a port to the JWKS, or look in
+   the server's ``/.well-known/jwks.json`` file. `RFC 5785
+   <https://tools.ietf.org/html/rfc5785>`_ describes the ``.well-known``
+   mechanism.
+
+   .. caution::
+
+        The Trino server also accepts JWKS URLs that begin with ``http://``, but
+        using this protocol results in a severe security risk. Only use this
+        protocol for short-term testing during development of your cluster.
+
+-  The path to a local file in :doc:`PEM </security/inspect-pem>` or `HMAC
+   <https://en.wikipedia.org/wiki/HMAC>`_ format that contains one or more
+   certificates that validate the Trino coordinator.
+
+   If the file contains more than one certificate, you can use ``$KEYID`` as
+   part of the path name as a placeholder, where KEYID is substituted at runtime
+   with the value of a JWT's ``kid`` field to identify a particular token.
+
+Trino CLI arguments
+-------------------
+
+The Trino :doc:`CLI </installation/cli>` provides the command line argument
+``--access-token=<token>`` to allow the CLI to be used with a custom
+authentication system based on JWT tokens.
+
+This presumes that the custom authenticaion system provides a way to extract a
```

Comment:
authenticaTion (2nd t is missing)

Reply:
Fixed. Unless I actually meant to use authenticaion, a new technology just coming out. Now you'll lose sleep over this, huh?